app/renderer/js/utils/link-util.js

```diff
@@ -21,6 +21,13 @@ class LinkUtil {
 
 		return (currentDomain === newDomain) && newUrl.includes('/#narrow');
 	}
+
+	isImage(url) {
```

Comment:
This should be `isImageURL` right?
cc @akashnimare @abhigyank

Reply:
It is already a `LinkUtil` file, so that part is understoond I suppose. I wrote it similar to already existing `isInternal`.

Reply:
Ah, yeah makes sense. `LinkUtil.isImage` sounds good (but  `LinkUtil.isImageURL` also does); You can ignore this then.